allensdk/brain_observatory/nwb/__init__.py

```diff
@@ -0,0 +1,66 @@
+from pynwb.base import TimeSeries
+from pynwb.behavior import BehavioralTimeSeries
+
+def add_running_speed_to_nwbfile(nwbfile, running_df, unit_dict):
```

Comment:
rename to `add_running_data_df_nwbfile`

Reply:
this will change to accept a dataframe with timestamps as index

Reply:
Done.  Also changed "S" to "s" for seconds